simulator/src/test/java/com/hazelcast/simulator/probes/impl/HdrProbeTest.java

```diff
@@ -26,16 +30,19 @@ public void testConstructor_noThroughputProbe() {
         assertFalse(tmpProbe.isPartOfTotalThroughput());
     }
 
+    // this test is fragile and can easily fail if there is some kind of pausing going on
```

Comment:
Of course it can happen that you have to change a custom assert method, if you change the production code. And ugly or not, it worked locally and on CloudBees (I can't remember any random failures with these tests).

But I don't care, as long as the new validation works. But after merging this I got a constant failure of `HdrProbeTest.testDone_withExternalStarted()` when run locally:
```
java.lang.AssertionError: Value:2000000000 not found in histogram

	at org.junit.Assert.fail(Assert.java:88)
	at com.hazelcast.simulator.probes.impl.HdrProbeTest.assertHistogramContent(HdrProbeTest.java:87)
	at com.hazelcast.simulator.probes.impl.HdrProbeTest.testDone_withExternalStarted(HdrProbeTest.java:46)
```